deploy/vagrant/create

```diff
@@ -176,17 +176,18 @@ fi
 masters=($(echo $masters | tr '\n' ' ' | tr -d '\r'))
 
 if [[ "$TACHYON_MASTERS" -gt 1 ]]; then
-  info ">>> Tachyon is in Fault Tolerant mode, all Tachyon masters: <<<"
+  info "Tachyon is in Fault Tolerant mode, all Tachyon masters:"
+  for master in ${masters[@]}; do
+    info "Tachyon master: ${master}"
+    info "Trying to curl Tachyon web UI at ${master}:19999"
+    curl "${master}:19999" >/dev/null
+    if [[ "$?" == "0" ]]; then
+      info "Master public IP for Tachyon is ${master}"
+      break
+    fi
+  done
+  info "Master public IP for other software is ${main_master}"
+else
+  info "TachyonMaster public IP is ${main_master}"
```

Comment:
I think we should just print the IP here, since the IP is also for other masters.